src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -129,7 +169,8 @@ public static class EditPersonDescriptor {
         private Phone phone;
         private Email email;
         private Address address;
-        private Set<Tag> tags;
+        private Set<Index> eventIndexTags;
+        private Set<EventTag> tags;
```

Comment:
I think the `tags` variable name might cause misunderstandings since we have `eventIndexTags`. Might have to update the `setTags` method name too.

```suggestion
        private Set<EventTag> eventTags;
```